packages/esm-patient-chart-app/src/encounter-list/components/tag.component.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Tag } from '@carbon/react';
+import { getObsFromEncounter, findObs } from '../../clinical-views/utils/helpers';
+
+export const renderTag = (encounter, concept, statusColorMappings) => {
```

Comment:
Please add types here.

Reply:
Converting it to a component becomes a bit hard to return it in a nested function